documentation/slate/source/includes/apis.md

```diff
@@ -1716,6 +1716,168 @@ object. Arrivals are ordered chronologically in ascending order.
 
 they are exactly the same as [departures](#departures)
 
+<a name="line-reports"></a>Line reports
+---------------------------------------
+``` shell
+#request
+$ curl 'http://api.navitia.io/v1/coverage/sandbox/line_reports' -H 'Authorization: 3b036afe-0110-4202-b9ed-99718476c2e0'
+
+#response, composed by 2 main lists: "line_reports" and "disruptions"
+HTTP/1.1 200 OK
+
+{
+"disruptions": [
+        #list of linked disruptions
+],
+"line_reports": [
+    {
+        "line": {
+            #main object (line) and links within its own disruptions
+        }
+        "pt_objects": [
+            #list of all disrupted objects related to the line: stop_area, networks, etc...
+        ]
+    },
+    {
+        #Another line with its objects
+    }
+]
+}
+
+```
+
+This service provides the state of public transport traffic, grouped by lines and all their stops. 
+It can be called for an overall coverage or for a specific object.
+
+<img src="./images/traffic_reports.png" alt="Traffic reports" width="300"/>
+
+### Parameters
+
+You can access it via that kind of url: <https://api.navitia.io/v1/{a_path_to_a_resource}/line_reports>
+
+For example:
+
+-   overall public transport line report on Ile de France coverage
+    -   <https://api.navitia.io/v1/coverage/fr-idf/line_reports>
+-   Is there any perturbations on the RER network ?
+    -   <https://api.navitia.io/v1/coverage/fr-idf/networks/network:RER/line_reports>
+-   Is there any perturbations on the "RER A" line ?
+    -   <https://api.navitia.io/v1/coverage/fr-idf/networks/network:RER/lines/line:TRN:DUA810801043/line_reports>
+
+
+The response is made of an array of [line_reports](#line-reports),
+and another one of [disruptions](#disruption).
+
+There are inner links between this 2 arrays:
+see the [inner-reference](#inner-references) section to use them.
```

Comment:
This is the parameter section, and it doesn't talk about parameter.

There is nothing about disruptions parameters that are quite useful for this API (I think about since and until)

Reply:
:heavy_check_mark: Done, there is now a table for params